src/integrationtest/python/steps/step_resource.py

```diff
@@ -0,0 +1,25 @@
+
+from PyQt5.QtTest import QTest
+from PyQt5.QtCore import Qt
+
+from behave import use_step_matcher, when, then
+
+from test.steps.step_helpers import assert_equal, assert_is_not, assert_is_true, assert_is_not_true
```

Comment:
this looks to make stuff fail:

```
  File "src/integrationtest/python/steps/step_create_account.py", line 8, in <module>
    from test.steps.step_helpers import assert_equal, assert_is_not, assert_is
ModuleNotFoundError: No module named 'test.steps'
```

changing it to `src.integrationtest.python.steps` works for me

Reply:
(copied from above)